packages/manager/apps/pci-workflow/src/api/hooks/order.ts

```diff
@@ -42,3 +50,56 @@ export const useInstanceSnapshotPricing = (
     data: snapshotPricing,
   };
 };
+
+export const findLocalZoneInstanceSnapshotPricing = (
```

Comment:
Same here, as you are calling `useQueries` hook inside, so this function should be a hook.